doc/developers/develop.rst

```diff
@@ -475,9 +475,16 @@ on a classifier, but not otherwise. Similarly, scorers for average precision
 that take a continuous prediction need to call ``decision_function`` for classifiers,
 but ``predict`` for regressors. This distinction between classifiers and regressors
 is implemented using the ``_estimator_type`` attribute, which takes a string value.
-It should be ``"classifier"`` for classifiers and ``"regressor"`` for
-regressors and ``"clusterer"`` for clustering methods, to work as expected.
-Inheriting from ``ClassifierMixin``, ``RegressorMixin`` or ``ClusterMixin``
+This attribute should have the following values to work as expected:
+
+- ``"classifier"`` for classifiers
+- ``"regressor"`` for regressors
+- ``"clusterer"`` for clustering methods
+- ``"outlier_detector"`` for outlier detectors.
+- ``"DensityEstimator"`` for density estimators.
+
+Inheriting from ``ClassifierMixin``, ``RegressorMixin``, ``ClusterMixin``,
+``DensityEstimator`` or ``OutlierMixin``,
```

Comment:
```suggestion
Inheriting from :class:`~base.ClassifierMixin`, :class:`~base.RegressorMixin`,
:class:`~base.ClusterMixin`, :class:`~base.OutlierMixin`, or :class:`~base.DensityMixin`
```

I think it would be nice to link to those mixin classes. Also I think `DensityEstimator` is a typo - it should be `DensityMixin`?

Reply:
Thanks for the feedback. Corrected.